doc/dev/developer_guide/tests-integration-tests.rst

```diff
@@ -54,9 +54,7 @@ Nightlies are often half-finished and cancelled due to the volume of testing
 done so your jobs may never finish. Therefore, it is common to select a
 priority less than 1000.
 
-Any priority may be selected when submitting jobs. But, in order to be
-sensitive to the workings of other developers that also need to do testing,
-the following recommendations should be followed:
+Job priority should be selected based on the following recommendation:
```

Comment:
Sorry, since there are more than one recommendation, we need to use plural form, shall we?

Reply:
I have updated this one too.